zql/docs/processors/README.md

```diff
@@ -113,6 +95,33 @@ TIME                        UID
 ...
 ```
 
+---
+
+## `drop`
+
+|                           |                                                             |
+| ------------------------- | ----------------------------------------------------------- |
+| **Description**           | Return the data only from all but the specified named fields.       |
+| **Syntax**                | `drop <field-list>`                   |
+| **Required<br>arguments** | `<field-list>`<br>One or more comma-separated field names or assignments.  |
+| **Developer Docs**        | https://pkg.go.dev/github.com/brimsec/zq/proc/cut            |
```

Comment:
```suggestion
| **Developer Docs**        | https://pkg.go.dev/github.com/brimsec/zq/proc            |
```

These links are a mild inconvenience: It'd return a 404 until we cut a GA release and tell pkg.go.dev to fetch the docs for that latest tag. For now we can link to a spot one level up and I've made a note-to-self to push an update for this link right after I post the next release.